### ISSUE TITLE
Loadbalancer does not start

### ISSUE THREAD

**Issue by slaskawi** [bsingr/keycloak-docker-jdbcping-cluster-example#1]
Hey!

Thanks for a great reproducer! I managed to reproduce this issue, however, there seems to be a permission issue on Loadbalancer service:

```
loadbalancer_1    | 2019/03/22 08:28:47 [emerg] 1#1: open() "/etc/nginx/conf.d/default.conf" failed (13: Permission denied) in /etc/nginx/nginx.conf:31
loadbalancer_1    | nginx: [emerg] open() "/etc/nginx/conf.d/default.conf" failed (13: Permission denied) in /etc/nginx/nginx.conf:31
```

Probably some `chmod` magic should do the trick.

**Comment by bsingr** [comment]
@slaskawi welcome.

Just that I understand correctly, you could reproduce the actual issue with "An error occurred, please login again through your application." etc.?

The problem you have with the nginx thing is not related to that and you worked around it somehow, right? For me it works (using a macOS docker host).

**Comment by slaskawi** [comment]
Yes, I managed to work it out without a load balancer. I basically logged into one instance and inspected Infinispan/JGroups using CLI.

I'll soon add a comment to the ticket.

**Comment by bsingr** [comment]
Ok, got it.
